src/puripuly_heart/ui/app.py

```
@@ -1090,6 +1102,85 @@ def _close_discord_managed_auth_dialog(self) -> None:
+                return
+            try:
+                ok = await start_auth(
+                    qq_identity=qq_identity,
+                    credential=credential,
+                )
+                if not ok:
+                    return
+                enable_translation = getattr(controller, "set_translation_enabled", None)
+                if not callable(enable_translation):
+                    return
+                enable_result = await enable_translation(True)
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)